VoxelSniperCore/src/main/java/com/github/kevindagame/util/Shapes.java

```diff
@@ -147,12 +145,12 @@ public static Set<BaseLocation> dome(BaseLocation location, int brushSize, int h
         return positions;
     }
 
-    public static Set<BaseLocation> line(VoxelVector originCoords, VoxelVector targetCoords, IWorld world) {
-        final VoxelVector HALF_BLOCK_OFFSET = new VoxelVector(0.5, 0.5, 0.5);
-        final VoxelVector originClone = originCoords.clone().add(HALF_BLOCK_OFFSET);
-        final VoxelVector targetClone = targetCoords.clone().add(HALF_BLOCK_OFFSET);
+    public static List<BaseLocation> line(VoxelVector originCoords, VoxelVector targetCoords, IWorld world) {
+        final VoxelVector HALF_BLOCK_OFFList = new VoxelVector(0.5, 0.5, 0.5);
```

Comment:
whoops lol (also change in places where it is used ofcourse)
```suggestion
        final VoxelVector HALF_BLOCK_OFFSET = new VoxelVector(0.5, 0.5, 0.5);
```